src/script/hydra-eval-jobset

```diff
@@ -357,22 +357,21 @@ sub evalJobs {
     my @cmd;
 
     if (defined $flakeRef) {
-        @cmd = ("hydra-eval-jobs",
-                "--flake", $flakeRef,
-                "--gc-roots-dir", getGCRootsDir,
-                "--max-jobs", 1);
+        @cmd = ("nix-eval-jobs", "--flake", $flakeRef . '#hydraJobs');
```

Comment:
Uses `hydraJobs` here instead of `flakeRef` and the original `hydra-eval-jobs` has `hydraJobs` and `checks`:

https://github.com/NixOS/hydra/blob/f974891c76e295240017dd7f04d50ecb4b70284e/src/hydra-eval-jobs/hydra-eval-jobs.cc#L148-L152

Reply:
Hmm?

Reply:
Ah found more commits that are relevant.

Reply:
The Lix commits don't address this issue, they only have support of hydraJobs.

Reply:
@Mic92 Do you have a preference for how we do this?

Reply:
Could we add an `nix eval` before that to figure out which one exists?

```
nix eval --raw --impure --expr "let f = builtins.getFlake \"git+file:$(pwd)\"; in if f ? hydraJobs then \"hydraJobs\" else \"checks\""
```

Reply:
Alternative is fixing: https://github.com/nix-community/nix-eval-jobs/pull/332
Than we can just pass in what we want.

Reply:
OK added a commit that ought to do this, but we should have some tests.